migration/migration_test.go

```diff
@@ -0,0 +1,71 @@
+package migration
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/almighty/almighty-core/configuration"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+)
+
+func TestConcurrentMigrations(t *testing.T) {
+	resource.Require(t, resource.Database)
+
+	var db1 *gorm.DB
```

Comment:
To increase the amount of concurrency, I suggest to use a loop to create more than just two database objects. Let's say 20. The example in `sync.WaitGroup` also does this in a loop: https://golang.org/pkg/sync/#WaitGroup.